airflow/www_rbac/security.py

```diff
@@ -170,6 +175,11 @@ class AirflowSecurityManager(SecurityManager, LoggingMixin):
             'perms': VIEWER_PERMS | USER_PERMS | OP_PERMS | DAG_PERMS,
             'vms': VIEWER_VMS | DAG_VMS | USER_VMS | OP_VMS,
         },
+        {
+            'role': 'Default',
+            'perms': VIEWER_PERMS | USER_PERMS | OP_PERMS,
+            'vms': VIEWER_VMS | USER_VMS | OP_VMS,
```

Comment:
Can we use the constants defined above?

```suggestion
            'perms': DFEAULT_PERMS,
            'vms': DEFAULT_VMS,
```

Reply:
Agree with this.